src/TodoList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import TodoListItem from "./TodoListItem";
+
+const TodoList = ({ todoList }) => {
+  return (
+    <>
+      <ul>
+        {todoList.map(function (todo) {
+          return <TodoListItem key={todo.id} title={todo.title} />;
```

Comment:
Pro tip:
 You can also destructure `todo` in the function argument to:
 `{id, title}` just like with the props. 

 Also, arrow functions are less verbose 😃